Src/VimCore/Interpreter_Interpreter.fs

```diff
@@ -807,6 +808,97 @@ type VimInterpreter
             if lineRange.Count > 1 then
                 _foldManager.CreateFold lineRange)
 
+    member x.RunNormal (lineRange: LineRangeSpecifier) input =
+        let transactionMap = System.Collections.Generic.Dictionary<ITextBuffer, ILinkedUndoTransaction>();
+        let modeSwitchMap = System.Collections.Generic.Dictionary<ITextBuffer, IVimBuffer>();

```

Comment:
These maps should use an `ITextView` as the key instead of `ITextBuffer`.  It is possible, although typically unlikely in VIsualStudio, that two `IVimBuffer` instances will map to the same `ITextBuffer`.  The underlying WPF model allows for that to happen.  The `ITextView` to `IVimBuffer` mapping though is unique.